pkgs/applications/video/electronplayer/electronplayer.nix

```diff
@@ -0,0 +1,32 @@
+{ appimageTools, stdenv, fetchurl }:
+let
+  pname = "electronplayer";
+  version = "2.0.8";
+  name = "${pname}-${version}";
+
+  src = fetchurl {
+    url = "https://github.com/oscartbeaumont/ElectronPlayer/releases/download/v${version}-rc4/${name}.AppImage";
```

Comment:
weird, that rc4 is part of the url, although this is a stable release. But seems to be an upstream issue. This will probably change on the next stable release and the rc4 should be removed here then. Maybe you can add a comment explaining that?

Reply:
I'm not sure I completely understand, I should add a comment to remind the other maintainers / me to remove the -rc4 for the next releases, or do you want me to downgrade the version to 2.0.7 as the appimage was released with not additionals tags than 2.0.7 ?

Reply:
yes, just a reminder to remove the -rc4 if someone just bumps the version should be fine. since 2.0.8 seems to be proper release there is no need to downgrade to 2.0.7

Reply:
No problem, is it good ?

Reply:
looks good